<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_pack_dialog</class>
 <widget class="QDialog" name="new_pack_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pack_name_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pack_name_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pack_title_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>291</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pack_title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack Title </string>
   </property>
  </widget>
  <widget class="QLabel" name="pack_name_label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>English letters, digits and underscores only.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>new_pack_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>new_pack_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
